toolbox

```diff
@@ -1747,6 +1755,35 @@ case $op in
         fi
         exit
         ;;
+    run )
+        if is_integer "$podman_pid"; then
+	    # shellcheck disable=SC2119
+            forward_to_host
+        else
+            while has_prefix "$1" -; do
+                case $1 in
+                    -c | --container )
+                        shift
+                        exit_if_missing_argument --container "$1"
+                        toolbox_container=$1
+                        ;;
+                    -r | --release )
+                        shift
+                        exit_if_missing_argument --release "$1"
+                        arg=$(echo "$1" | sed "s/^F\|^f//" 2>&3)
+                        exit_if_non_positive_argument --release "$arg"
+                        release=$arg
+                        ;;
+                    * )
+                        exit_if_unrecognized_option "$1"
+                esac
+                shift
+            done
+            update_container_and_image_names
+            run "$*"
```

Comment:
`"$@"` would be [better](https://unix.stackexchange.com/questions/129072/whats-the-difference-between-and).